reporters/reporter-default/src/main/java/org/radargun/reporting/html/TimelineChart.java

```diff
@@ -97,7 +97,7 @@ public void setEvents(List<? extends Timeline.Event> events, int slaveIndex, lon
       for (Timeline.Event event : events) {
          if (event instanceof Timeline.Value) {
             Timeline.Value value = (Timeline.Value) event;
-            int bucket = (int) ((event.timestamp - startTimestamp) * MAX_EVENT_VALUES / (endTimestamp - startTimestamp));
+            int bucket = (int) ((event.timestamp - startTimestamp) * (MAX_EVENT_VALUES-1) / (endTimestamp - startTimestamp));
```

Comment:
Yep, the fix is fine - `endTimestamp` is a max of all event timestamps so it's possible that `event.timestamp == endTimestamp` and it overflows due to `bucket == MAX_EVENT_VALUES`